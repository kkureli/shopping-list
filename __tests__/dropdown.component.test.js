import React from 'react';
import renderer from 'react-test-renderer';
import Dropdown from '../src/components/common/dropdown';

describe('dropdown component', () => {
  it('renders dropdown correctly', () => {
    const tree = renderer.create(<Dropdown />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
