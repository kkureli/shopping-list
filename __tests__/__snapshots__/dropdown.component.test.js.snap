// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dropdown component renders dropdown correctly 1`] = `
<RCTScrollView
  ItemSeparatorComponent={[Function]}
  contentContainerStyle={
    Array [
      Object {
        "backgroundColor": "rgb(242, 242, 242)",
      },
      Object {
        "backgroundColor": "rgb(242, 242, 242)",
        "borderColor": "silver",
        "borderRadius": 5,
        "borderWidth": 1,
        "marginTop": 5,
        "width": "100%",
      },
    ]
  }
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  style={
    Object {
      "maxHeight": 160,
      "width": "90%",
    }
  }
  viewabilityConfigCallbackPairs={Array []}
>
  <View />
</RCTScrollView>
`;
